leakcanary-android/src/main/java/com/squareup/leakcanary/ActivityRefWatcher.java

```diff
@@ -68,6 +80,11 @@ public static void install(Application application, RefWatcher refWatcher) {
   public ActivityRefWatcher(Application application, RefWatcher refWatcher) {
     this.application = checkNotNull(application, "application");
     this.refWatcher = checkNotNull(refWatcher, "refWatcher");
+    if (VERSION.SDK_INT >= VERSION_CODES.O) {
+      this.fragmentRefWatcher = new FragmentRefWatcher(refWatcher);
```

Comment:
lazy-web: does the support library impl only work on O+, too?

Reply:
It works everywhere, so we could choose to move support fragment watching into ActivityRefWatcher and then make FragmentRefWatcher O-only. (The callback class must be imported in an O-only class)

Reply:
Split.